src/tile/tile.scss

```diff
@@ -175,9 +175,7 @@
       }
 
       > .iui-name-label {
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
+        @include iui-line-clamp(1);
```

Comment:
Having `min-width: 0` on `iui-content` does the same and also works on IE.

Reply:
How does that even work?

Reply:
I changed the line-clamp mixin with proper fallback so it should be fixed now

Reply:
When using `flex` by default `min-width` is the same size as its content therefore it goes outside the bounds.
Maybe I am old fashioned but for single line ellipsis I still prefer `text-overflow: ellipsis` 😀

Reply:
There, this should satisfy your old fashioned needs 😄